go.mod

```diff
@@ -32,85 +32,161 @@ require (
 	github.com/scjalliance/comshim v0.0.0-20190308082608-cf06d2532c4e
 	github.com/serenize/snaker v0.0.0-20171204205717-a683aaf2d516
 	github.com/shirou/gopsutil v3.21.11+incompatible
-	github.com/sirupsen/logrus v1.8.1 // indirect
+	github.com/sirupsen/logrus v1.9.0 // indirect
 	github.com/stretchr/testify v1.8.1
 	github.com/theupdateframework/notary v0.6.1
 	github.com/tklauser/go-sysconf v0.3.10 // indirect
 	github.com/yusufpapurcu/wmi v1.2.2 // indirect
 	go.etcd.io/bbolt v1.3.6
-	go.opencensus.io v0.23.0
+	go.opencensus.io v0.24.0
 	golang.org/x/crypto v0.4.0
-	golang.org/x/exp v0.0.0-20221126150942-6ab00d035af9
+	golang.org/x/exp v0.0.0-20230202163644-54bba9f4231b
 	golang.org/x/image v0.5.0
 	golang.org/x/net v0.7.0
-	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
+	golang.org/x/sync v0.1.0
 	golang.org/x/sys v0.5.0
 	golang.org/x/text v0.7.0
-	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
-	google.golang.org/grpc v1.38.0
+	golang.org/x/time v0.1.0
+	google.golang.org/grpc v1.52.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/toast.v1 v1.0.0-20180812000517-0a84660828b2
 	howett.net/plist v0.0.0-20181124034731-591f970eefbb
 	software.sslmate.com/src/go-pkcs12 v0.0.0-20210415151418-c5206de65a78
 )
 
 require (
+	github.com/anchore/syft v0.73.0
 	github.com/kolide/systray v1.10.2
-	github.com/vmihailenco/msgpack/v5 v5.3.5
 )
 
 require (
 	github.com/BurntSushi/toml v1.1.0 // indirect
+	github.com/CycloneDX/cyclonedx-go v0.7.1-0.20221222100750-41a1ac565cce // indirect
```

Comment:
that's a lot of additionally dependency 🤔

Reply:
Yeah.... Any idea what it does to the resultant size? I don't love it, but it might just be the cost of `syft`

Reply:
I suppose that's due to the nature of syft, but yeah... hmmm

Reply:
It nearly doubles the size of the build =( 

18.1 MB to 30.3 MB on MacOS

Additionally, there is some issues with building for windows in CI https://github.com/kolide/launcher/actions/runs/4308325630/jobs/7514473491

Not sure if the juice is worth the squeeze here.

Reply:
Wow, I would have not expected such a large increase